types/svgo/index.d.ts

```diff
@@ -10,6 +10,8 @@
 //                 Petr Zahradník <https://github.com/petrzjunior>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+/// <reference types="node"/>
```

Comment:
Is this required for `Buffer`? I’m seeing other packages omit it. For example, [ref uses `Buffer` but doesn’t use `reference`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/aa977cdb8b5120ae9dbe128b61356335cb071645/types/ref/index.d.ts).

Reply:
Yes, this is for `Buffer`.

Just clicked the link you gave. Hm.. Line number 6 is `/// <reference ...`. Or I misunderstood something?

First I was trying to add reference to Node’s types inside `tsconfig.json`, but test did not pass. Checked other libraries and saw triple-slash directive used. It made sense, of course. Might be there are other solutions.

Reply:
Huh, I could have sworn that wasn’t there before, but I guess I mixed up my tabs. Sorry about that! 😳 You’re absolutely right.